cmd/autoscaler/main.go

```diff
@@ -105,6 +107,18 @@ func main() {
 	defer flush(logger)
 	ctx = logging.WithLogger(ctx, logger)
 
+	kubeClient := kubeclient.Get(ctx)
```

Comment:
Let's move this block above the logger creation as we do in other components or use the "proper" logger created.

Reply:
Thank you. Yes, I moved the block above the logger creation which is same with activator.